pkg/descheduler/pod/pods_test.go

```diff
@@ -154,3 +157,27 @@ func TestSortPodsBasedOnPriorityLowToHigh(t *testing.T) {
 		t.Errorf("Expected last pod in sorted list to be %v which of highest priority and guaranteed but got %v", p4, podList[len(podList)-1])
 	}
 }
+
+func TestSortPodsBasedOnAge(t *testing.T) {
+	podList := make([]*v1.Pod, 9)
+	n1 := test.BuildTestNode("n1", 4000, 3000, int64(len(podList)), nil)
+
+	for i := 0; i < len(podList); i++ {
+		podList[i] = test.BuildTestPod(fmt.Sprintf("p%d", i), 1, 32, n1.Name, func(pod *v1.Pod) {
+			creationTimestamp := metav1.Now().Add(time.Minute * time.Duration(i))
+			pod.ObjectMeta.SetCreationTimestamp(metav1.NewTime(creationTimestamp))
+		})
+	}
+
+	rand.Seed(time.Now().UnixNano())
+	rand.Shuffle(len(podList), func(i, j int) { podList[i], podList[j] = podList[j], podList[i] })
```

Comment:
Nit: I wonder what's the percentage of cases where the shuffled list is identical to its original ordering.

Reply:
I think it should be factorial so pretty low. If my calculation is correct `1 / 40320`

But I can refactor it in another PR

Reply:
Addressed in: https://github.com/kubernetes-sigs/descheduler/pull/856